substrate/client/rpc-servers/src/lib.rs

```diff
@@ -52,6 +52,14 @@ const MEGABYTE: u32 = 1024 * 1024;
 /// Type alias for the JSON-RPC server.
 pub type Server = jsonrpsee::server::ServerHandle;
 
+/// Type to encapsulate the server handle and listening address.
+pub struct ServerAndListenAddress {
+	/// Handle to the rpc server
+	pub handle: Server,
```

Comment:
I would rather add getters for handle and listen_addr than to make this public to avoid breaking changes in the future....

Reply:
Done.